portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['A link is only as long as your longest strong chain', 'Search Warranty', 'Split it 50/50/50', 'Keep your friends close, but your enemies toaster'];
```

Comment:
As mentioned before, lets try to make this a consistent line length. ie.
      ['A link is only as long as your longest strong chain', 'Search Warranty',
       'Split it 50/50/50', 'Keep your friends close, but your enemies toaster'];

Reply:
As mentioned, I don't think I see this changed either.